ELIPs/ELIP-002.md

```diff
@@ -89,6 +93,41 @@ This upgrade adds new protocol interfaces and primitives for Operator Sets, Uniq
 
 These are provided through the introduction of the `AllocationManager` contract and changes to the `DelegationManager` to make use of the Unique Security model. The AVS middleware contracts are also updated. First, we will describe Operator Sets and how they give fine-grained, opt-in control of slashable Unique Stake to Operators and how they provide some key guarantees to AVSs.
 
+## AVS Metadata
```

Comment:
Move this section right before Creation, Registration, & Deregistration

and name it "Setting AVS Metadata"